Add presence for V2ex

MeXunco opened the pull request:
[V2EX](https://www.v2ex.com) is a community of start-ups, designers, developers and creative people for china.

<details>
  <summary>Click here to view screenshots</summary>
  
  ![](https://i.imgur.com/gCajS53.png)
  ![](https://i.imgur.com/IE5LTZw.png)
  ![](https://i.imgur.com/Ab9IGMD.png)
  ![](https://i.imgur.com/tsxFd8M.png)
  ![](https://i.imgur.com/XOwqXMn.png)
</details>

Alanexei commented on V2ex/dist/metadata.json:
```diff
@@ -0,0 +1,21 @@
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/kfj4PUe.png",
+  "thumbnail": "https://i.imgur.com/eugm5S4.png",
+  "color": "#000000",
```
This isn't necessary, but can you find a color that accustomed to the website's theme? Such as a dark blue or dark gray.

MeXunco replied:
Thanks, please wait.

Bas950 commented:
> ![image](https://user-images.githubusercontent.com/34792838/79289099-b4cb4200-7ec8-11ea-92f8-bda1fe5c4063.png)
> Delete this file

Your presence right now is:
presence
├── dist
│   ├── metadata.json
│   └── presence.js
├── presence.ts
├── presence.js
└── tsconfig.json

It should be:
presence
├── dist
│   ├── metadata.json
│   └── presence.js
├── presence.ts
└── tsconfig.json

MeXunco commented:
I have changed that, thank you for pointing that out!



> > ![image](https://user-images.githubusercontent.com/34792838/79289099-b4cb4200-7ec8-11ea-92f8-bda1fe5c4063.png)
> > Delete this file
> 
> Your presence right now is:
> presence
> ├── dist
> │ ├── metadata.json
> │ └── presence.js
> ├── presence.ts
> ├── presence.js
> └── tsconfig.json
> 
> It should be:
> presence
> ├── dist
> │ ├── metadata.json
> │ └── presence.js
> ├── presence.ts
> └── tsconfig.json


I have changed that, thank you for pointing that out!

Bas950 commented:
:thonk:

MeXunco commented:
Sorry, I made a mistake👼

MeXunco commented:
It seems that you need to set up

> ERROR environment variable GITHUB_TOKEN not set! [click link](https://github.com/PreMiD/Presences/pull/1327/checks?check_run_id=590889675)

doomlerd commented:
@Fruxh
